lib/src/platform/src/push_activation_events_internal.dart

```diff
@@ -4,18 +4,22 @@ import 'dart:async';
 
 import 'package:ably_flutter/ably_flutter.dart';
 
+/// @nodoc
 /// package-private implementation of [PushActivationEvents]
```

Comment:
All comments in this file could be "upgraded" to sentences - starting with a capital letter and ending with the dot if we want to

Reply:
Done in 19b0814c30acaf767beee3ed6eecd362b672f012